assets/Product.wxs

```diff
@@ -10,14 +10,16 @@
   <?define ProductName = "$(env.ProductName)" ?>
   <?define ProductGuid = "$(env.ProductGuid)" ?>
   <?define ProductVersion = "$(env.ProductVersion)" ?>
+  <?define ProductSemanticVersion = "$(env.ProductSemanticVersion)" ?>
   <?define ProductVersionWithName = "$(var.ProductName)_$(var.ProductVersion)"?>
+  <?define ProductSemanticVersionWithName = "$(var.ProductName)_$(env.ProductSemanticVersion)"?>
```

Comment:
Seems like we're inconsistent with underscores and dashes.  Unless there's a reason to use underscore here, perhaps we should standardize on dashes?